src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
-}
+  const obj = {};
```

Comment:
`obj` is not the best name for the copied robot

Reply:
I will take this into consideration Misha!